skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillModel.java

```diff
@@ -5,16 +5,16 @@
 import com.fasterxml.jackson.annotation.JsonBackReference;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import jakarta.persistence.*;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
+import lombok.*;
 
 import java.time.Instant;
 import java.util.Set;
 import java.util.UUID;
 
 @EqualsAndHashCode(callSuper = true)
 @Entity
+@Builder
```

Comment:
Same idea with Builder/AllArgs/NoArgs here. Check if this is needed and remove if its not necessary.

Reply:
done updated that